package/MDAnalysis/analysis/psa.py

```diff
@@ -218,6 +218,7 @@
 
 import numpy as np
 from scipy import spatial, cluster
+from scipy.spatial.distance import directed_hausdorff
```

Comment:
what is the minimum scipy version we depend on now?

Reply:
Shouldn't be hard to check -- maybe making it the "stable" 1.0 version would be sensible soon.

Reply:
This function entered scipy in 0.19.0 according to the docs. So I think we should raise our minimum required version to this one.

Reply:
@kain88-de thanks for catching this!